app/models/authentication.rb

```diff
@@ -2,7 +2,7 @@ class Authentication < ApplicationRecord
   acts_as_miq_taggable
   include_concern 'ImportExport'
   include YAMLImportExportMixin
-
+  include SupportsFeatureMixin
```

Comment:
Should be able to drop the include from the cloud_manager auth_key_pair now https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/cloud_manager/auth_key_pair.rb#L2